tests/unit/gateways/clients/test_wallet_service_db_client.py

```diff
@@ -54,17 +54,23 @@ def test_get_address_history(self, engine, connection):
         address = "H" + fake.pystr()
         token = fake.sha256()
         limit = fake.random_int()
-        offset = fake.random_int()
+        last_ts = fake.random_int()
+        last_tx = fake.pystr()
```

Comment:
```suggestion
        last_tx = fake.sha256()
```

It shouldn't change anything but it sends a value closer to real data.